### PR TITLE
Support for presigned urls in GET /bundles.

### PR DESCRIPTION
**Pull request by ttung**

### CODE REVIEW

File: daemons/package_daemon.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ daemon=$1
 
 echo "Running pre-packaging steps for $daemon"
 
-git clean -df $daemon/domovoilib $daemon/vendor
+#git clean -df $daemon/domovoilib $daemon/vendor
```
Comment (mweiden) [inline daemons/package_daemon.sh]: ?

**Comment by ttung** [comment]
Updated.

**Comment by ttung** [comment]
> Please add a test where both URL types are requested.

um wat?

https://github.com/HumanCellAtlas/data-store/pull/1345/files#diff-307f004e5f5806557cc669e52b0cdd1eR147

**Comment by Bento007** [comment]
@ttung you're too fast

**Pull request merged**